presto-elasticsearch/pom.xml

```diff
@@ -300,10 +300,15 @@
         </dependency>
 
         <!-- for testing -->
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>testcontainers</artifactId>
+            <scope>test</scope>
+        </dependency>
+
```

Comment:
the dependency check fails at this is used and not provided

Reply:
Ah, that's because TestElasticsearchIntegrationSmokeTest is importing `org.testcontainers.shaded.com.fasterxml.jackson.databind.ObjectMapper`. See my comment below.

Reply:
fixed